### PR TITLE
Add `bvd_id` to company search response.

### PR DESCRIPTION
**Pull request by Diggsey**

### CODE REVIEW

**Review by louisdewar — APPROVED** [review]
Looks fine, you might want to ping a technical writer to confirm the language. Also if I was Richard I would tell you to put the `&apos;` changes for the pre-existing content into its own commit

File: source/includes/_08-additional-features.md.erb [hunk 1]
```diff
@@ -8,7 +8,7 @@ In order to run a successful company data check a company must be uniquely
 identified. Some company data providers allow searching through the candidates
```
Comment (AinsleyWhalen) [inline source/includes/_08-additional-features.md.erb:8]: Add comma: "In order to run a successful company data check,"

File: source/includes/_08-additional-features.md.erb [hunk 2]
```diff
@@ -68,7 +68,7 @@ The payload of the request can contain the following fields
       <td>Yes</td>
       <td>
         This field contains the provider config structured in the way
-        specified by your integration's
```
Comment (AinsleyWhalen) [inline source/includes/_08-additional-features.md.erb:71]: Change "config" to "configuration"

File: source/includes/_08-additional-features.md.erb [hunk 3]
```diff
@@ -309,10 +309,10 @@ app.post("/search", (req, res) => {
         </ul>
         
         For demo searches, charges may be returned, but they will not be deducted
```
Comment (AinsleyWhalen) [inline source/includes/_08-additional-features.md.erb:311]: Remove comma: "...charges may be returned but they will not..."

**Review by AinsleyWhalen — APPROVED** [review]
Only a few small typos with the old text. No changes required for the new text.

File: source/includes/_08-additional-features.md.erb (hunk 1 above)
Comment (Diggsey) [reply source/includes/_08-additional-features.md.erb:8]: 👍

File: source/includes/_08-additional-features.md.erb (hunk 2 above)
Comment (Diggsey) [reply source/includes/_08-additional-features.md.erb:71]: 👍

File: source/includes/_08-additional-features.md.erb (hunk 3 above)
Comment (Diggsey) [reply source/includes/_08-additional-features.md.erb]: 👍

**Pull request merged**